changelog/v1.17.0-rc5/update-envoy-repo-loc.yaml

```diff
@@ -0,0 +1,3 @@
+changelog:
+  - type: NON_USER_FACING
```

Comment:
for anyone following along, this changelog should have been v1.18.0-rc1 (we bumped to envoy 1.18)

we will fix and do another release in follow up @saiskee @EItanya 

this release has already been marked as a pre release